tree/ntuple/v7/src/RColumn.cxx

```diff
@@ -46,7 +46,9 @@ void ROOT::Experimental::Detail::RColumn::Connect(DescriptorId_t fieldId, RPageS
    case EPageStorageType::kSink:
       fPageSink = static_cast<RPageSink*>(pageStorage); // the page sink initializes fHeadPage on AddColumn
       fHandleSink = fPageSink->AddColumn(fieldId, *this);
-      fHeadPage = fPageSink->ReservePage(fHandleSink);
+      fApproxNElementsPerPage = fPageSink->GetWriteOptions().GetApproxUnzippedPageSize() / fElement->GetSize();
```

Comment:
What if `fElement->GetSize() > fPageSink->GetWriteOptions().GetApproxUnzippedPageSize()`? Consider adding a test for this case?

Reply:
Yep, that should now throw on construction and is tested